.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: ci
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Does it make sense to execute the action if we then not execute any job? Because of ` if: startsWith(github.ref, 'refs/tags/v')`

Reply:
```suggestion
```